src/Oci8/Schema/Sequence.php

```diff
@@ -27,19 +27,18 @@ public function __construct(Connection $connection)
      * @param  bool $nocache
      * @return bool
      */
-    public function create($name, $start = 1, $nocache = false)
+    public function create($name, $start = 1, $nocache = false, $min = 1, $max = 10000, $increment = 1)
```

Comment:
`$max = 10000` seems to low. Maybe a better approach is to set `$max = false`? And if it's not set, we will not include `maxvalue $max` on the sql.